tests/utils.go

```diff
@@ -2906,17 +2906,20 @@ func configureIPv6OnVMI(vmi *v1.VirtualMachineInstance, expecter expect.Expecter
 		return err == nil
 	}
 
-	dnsServerIP, err := getClusterDnsServiceIP(virtClient)
-	if err != nil {
-		log.DefaultLogger().Object(vmi).Infof("Couldn't get DNS Service IP while configuring ipv6: %v", err)
-		expecter.Close()
-		return err
+	clusterSupportsIpv6 := func(vmi *v1.VirtualMachineInstance) bool {
```

Comment:
Why do you keep it as an argument while you've dropped it from all the others?

Reply:
here it's actually used - `pod := GetPodByVirtualMachineInstance(vmi, vmi.Namespace)`

Reply:
but yes, we can drop it from here as well

Reply:
Please, do. The rest is +1 from me

Reply:
done